src/hotspot/share/opto/replacednodes.cpp

```diff
@@ -222,6 +222,7 @@ void ReplacedNodes::apply(Compile* C, Node* ctl) {
     }
 
     // Clone nodes and record mapping from current to cloned nodes
+    uint old_nodes = C->unique();
```

Comment:
I suggest the following name instead to better describe what the uint is:
```suggestion
    uint idx_before_clone = C->unique();
```